src/Extension/FluentVersionedExtension.php

```diff
@@ -151,6 +151,15 @@ protected function augmentDatabaseRequireTable($localisedTable, $fields, $indexe
         DB::require_table($localisedTable . FluentVersionedExtension::SUFFIX_VERSIONS, $versionsFields, $versionsIndexes, false);
     }
 
+    protected function isFieldLocalised($field, $type, $class)
```

Comment:
This might have impact on dev/build, note to self: need to cover this by detailed locale testing.